web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -326,6 +342,41 @@ export default class AccessGrants extends Vue {
     public get selectedAccessGrantsAmount(): number {
         return this.$store.state.accessGrantsModule.selectedAccessGrantsIds.length;
     }
+
+    /**
+     * Access grant button click.
+     */
+    public accessGrantClick(): void {
+        this.modalAccessType = 'access';
+        this.toggleAccessModal();
+    }
+
+    /**
+     * S3 Access button click..
+     */
+    public s3Click(): void {
+        this.modalAccessType = 's3';
+        this.toggleAccessModal();
+    }
+
+    /**
+     * CLI Access button click.
+     */
+    public cliClick(): void {
+        this.modalAccessType = 'api';
+        this.toggleAccessModal();
+    }
+
+    /**
+     * toggles Create Access Modal visibility.
+     */
+    public toggleAccessModal(): void {
+        if (this.showAccessModal === false) {
+            this.showAccessModal = true;
+        } else {
+            this.showAccessModal = false;
+        }
```

Comment:
this could be simplified to 
this.showAccessModal = !this.showAccessModal

Reply:
Will be implemented in next commit.